public/index.html

```diff
@@ -6,6 +6,7 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
```

Comment:
`Helvitica` was defined as the font in `Main.scss`. Which one should we go with?

Reply:
I have used Roboto for some headings. I believe this is inevitable since it might not be possible to have a single font for the whole application.